std/c/netbsd.zig

```diff
@@ -42,14 +42,12 @@ pub const pthread_attr_t = extern struct {
 };
 
 pub const msghdr = extern struct {
-    msg_name: *u8,
+    msg_name: ?*c_void,
```

Comment:
I think this would be better as `[*]u8`. It's the same ABI. The question is whether or not the address 0x0 is valid, for whether to have a `?`. Same thing for `iov_base` below.

Reply:
wouldn't there arise a scenario where msg_name might be assigned to null? same with iov_base

Reply:
In this case the type should be `?[*]u8`.